<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CAC4" ignored="0" name="BankingApp Test" tests="7" failures="3" timestamp="30 Nov 2018 03:12:36 GMT" time="123.938" errors="0">
  <testcase name="loginTest" time="1.563" classname="com.inetbanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="8.32" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.645" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="4.025" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="8.046" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="8.043" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="addNewCustomer" time="14.724" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- BankingApp Test -->
